docs/modules/ROOT/pages/how-to-guides/configuring-builds/proc_customize_build_pipeline.adoc

```diff
@@ -0,0 +1,99 @@
+= Customizing the build pipeline
+
+You can customize the way that {ProductName} builds components, to meet your specific needs, in two ways:
+
+* By changing the parameters of the build pipeline of a component
+* By extending the pipeline with your own tasks (or deleting existing tasks)
+
+For example, you can change for how long {ProductName} saves the container images of component, by changing the value of the `image-expires-after` parameter. Or you can ensure that the container images it builds for a component are compliant with regulations for your industry, by adding a compliance check as a task to its build pipeline. 
+
+.Prerequisites
+To customize a build pipeline, you must first xref:how-to-guides/configuring-builds/proc_upgrade_build_pipeline.adoc[upgrade it]. 
+
+.Procedure
+
+To customize the build pipeline in any way, start by completing the following steps:
+
+. In your preferred IDE, navigate to the `.tekton` directory in the repository of your component.
+
+. There should be two files, one ending with `push.yaml` and another ending with `pull-request.yaml`. Open both of them.
```

Comment:
Instead of saying what "should be" make it action oriented. 

Open two files. The file that ends with `push.yaml` and the file that ends with `pull-request.yaml`. 

If there are _not_ these files, we can include troubleshooting steps. If we don't have those steps handy, let's not even open the door with what "should" and "shouldn't" happen.